sagemathinc/cocalc
# Fix computed values

# Description
The computed values are run every time they're called at the moment. This PR fixes that.

# Testing Steps
1. `cd src/smc-util`
1. `npm test`
1. In the browser console `smc.redux.getProjectStore(project_id).selectors.stripped_public_paths.fn.recomputations();` with `project_id` appropriately filled in.
1.  Check any file
1. Run the same code above.
1. The recomputation count should not go up

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
